packages/cyberstorm/src/components/Button/Button.tsx

```diff
@@ -79,17 +83,21 @@ const Button = React.forwardRef<
     const fRef = forwardedRef as React.ForwardedRef<HTMLButtonElement>;
     const ref = fRef || fallbackRef;
     return (
-      <button
-        {...forwardedProps}
-        ref={ref}
-        type={type}
-        className={`${styles.root} ${getIconAlignment(
-          iconAlignment
-        )} ${getStyle(colorScheme)} ${getPaddingSize(paddingSize)}`}
-        onClick={onClick}
-      >
-        {children}
-      </button>
+      <TooltipProvider>
```

Comment:
`<TooltipProvider>` is defined in `packages\cyberstorm\src\components\Providers.tsx`, so this wrapper can be removed.

Reply:
Oh, ok. It exports delay when we call tooltip or do I still need to trigger it somehow?

Reply:
Nope, it's fine with just the Tooltip in use in the Button component. Like I said before, the provider is defined in the providers file and has to be initiated only once in the project (project either NextJS or storybook). If you feel a bit unclear after this comment; reading up on how providers work will clarify things up 👍